misc/src/androidTest/java/com/example/snippets/DeviceCompatibilityModeTestJavaSnippets.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.snippets;
+
+import android.app.Activity;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import org.junit.Rule;
+import static org.junit.Assert.assertFalse;
+
+public class DeviceCompatibilityModeTestJavaSnippets extends AppCompatActivity {
```

Comment:
Nit - I don't think any of these test classes need to subclass `Activity`, they can just be plain classes that don't extend anything.

Reply:
You are correct. I got too used to doing this for other snippets. Removed.